pkg/domain/entities/pods.go

```diff
@@ -106,23 +106,143 @@ type PodRmReport struct {
 	Id  string //nolint
 }
 
+// PodCreateOptions provides all possible options for creating a pod and its infra container
+// swagger:model PodCreateOptions
 type PodCreateOptions struct {
-	CGroupParent       string
-	CreateCommand      []string
-	Hostname           string
-	Infra              bool
-	InfraImage         string
-	InfraName          string
-	InfraCommand       string
-	InfraConmonPidFile string
-	Labels             map[string]string
-	Name               string
+	CGroupParent       string            `json:"cgroup-parent,omitempty"`
```

Comment:
@jwhonce PTAL I think this should be snake case to match the other json tags WDYT?

Reply:
Not used anymore since we do not decode into `PodCreateOptions` fixed